recipes-core/iotedge-cli/iotedge-cli_1.1.3.bb

```diff
@@ -0,0 +1,277 @@
+# Auto-Generated by cargo-bitbake 0.3.15
+#
+inherit cargo
+
+# If this is git based prefer versioned ones if they exist
+# DEFAULT_PREFERENCE = "-1"
+
+SRC_URI += "https://github.com/Azure/azure-iotedge/releases/download/${PV}/iotedge-${PV}.tar.gz"
+SRC_URI[md5sum]="88fc1f825b6285e92da0b635d033173a"
+SRC_URI[sha256sum]="c9e49a5243428b2b6edb5f2a6310ba45cd3d79067c0c6a2bfba9abab49082c3e"
```

Comment:
This is actually not the first time we have to manually maintain these hashes for each of the release -- we have to do the same thing for EFLOW RPM build. I wonder if we can do something during the release process to have this parameter be easily fetch (aka.ms link?) or a gitHub action that can maintain these hashes for us.